test/auth/int.spec.ts

```diff
@@ -307,6 +307,8 @@ describe('Auth', () => {
         },
       });
 
+      const body = await response.json();
```

Comment:
Is this doing anything? Were you planning to some assertions on `body`?

Reply:
Ah, this was leftover from me testing things out, will remove.